src/lbryio.js

```diff
@@ -110,17 +110,18 @@ Lbryio.authenticate = () => {
       Lbryio.getAuthToken()
         .then(token => {
           if (!token || token.length > 60) {
-            return false;
+            return [false];
           }
 
           // check that token works
           return Lbryio.getCurrentUser()
-            .then(() => true)
-            .catch(() => false);
+            .then(user => [true, user])
```

Comment:
Can't you just pass user or false?
User should never be false

Reply:
Ah that's a good point and much cleaner. I will update.